.github/workflows/docs.yaml

```
@@ -66,6 +66,8 @@ jobs:
             MDBOOK_OUTPUT__HTML__SITE_URL=/rsdk/$po_lang/ \
             MDBOOK_OUTPUT__HTML__REDIRECT='{}' \
             mdbook build -d book/html/$po_lang
+            cp -a book/html/$po_lang/html/* book/html/$po_lang/
+            rm -rf book/html/$po_lang/html
```

```suggestion
            mdbook build -d book/$po_lang
            mv book/$po_lang/html book/html/$po_lang
```
这样看看，也和make serve_zh-CN的输出路径保持一直